src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -1,3 +1,4 @@
+import {Platform} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Platform` from 'react-native' is a good addition to check the platform-specific code execution.
```diff
+ import {Platform} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->